sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/ReactiveCosmosTemplate.java

```diff
@@ -626,6 +668,26 @@ public <T> Mono<Void> deleteEntity(String containerName, T entity) {
         return deleteItem(originalItem, containerName, domainType).then();
     }
 
+    /**
+     * Delete all items with bulk.
+     *
+     * @param containerName the container name
+     * @param cosmosItemOperations the Flux of the CosmosItemOperation's to delete
+     * @return void Mono
+     */
+    public Mono<Void> deleteEntities(String containerName, Flux<CosmosItemOperation> cosmosItemOperations) {
```

Comment:
Does this need to be a public API?

Reply:
Since this is called from SimpleReactiveCosmosRepository it has to be public